layouts/partials/warning.ejs

```diff
@@ -0,0 +1,3 @@
+<div class="warning">
+  <%= message %>
```

Comment:
Since this template is named `warning`, would it make sense to include the `Warning:` as part of the template?

Reply:
👍